### DISCUSSION TITLE
[New Feature] Keep building repair on when low on cash

### DISCUSSION THREAD

**Discussion by MarkJFox** [Vinifera-Developers/Vinifera#1256] [Feature Requests]
### First Check

- [X] I added a very descriptive title to this issue.
- [X] I used the GitHub search to find a similar issue and didn't find it.
- [X] I have attached as much information as possible *(screenshots, videos, links, etc)*.

### Description

In TS, if you repair a building and you hit 0 credits, the repair job will instantly quit and you have to give the repair order again when credits are available. In modern RTS games, the repair job stays on while waiting for cash. If cash will be available again, the building repair continues automatically until the building is fully repaired. Then the repair job is done and has to be given again if a repair is needed. 

Maybe this could be activated via vinifera.ini like the tabbed sidebar.

**Comment by JoyfulShush** [comment]
For this, an icon that has a wrench and a "no access" across it would show that the repair is currently suspended due to insufficient funds to the repairing player and its allies.

For enemies, it should show no icon, as if the repair job was stopped, in order to not tip them for the fact that the player is broke. Once the repair job continues (thanks to players getting more funds), then enemies would see the wrench icon again, as if players clicked again on repair.
